cmd/debug/debug.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ignite/cli-plugin-airdrop/cmd"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "cli-plugin-airdrop",
+	Short: "debug command for CLI airdrop plugin",
+}
+
+func main() {
+	rootCmd.AddCommand(cmd.NewAirdrop())
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
+}
```

Comment:
Hey @Pantani, you don't really need these `debug` binary to test the plugin. As mentioned in the README, all you need to do is to declare the local path of the plugin in the configuration, and then run `ignite airdrop`. If you have updated the code, the plugin will be recompiled automatically.

Note that now you can use the `ignite` binary compiled from main, since the branch that was compliant with the latest plugin API has been merged.

Reply:
Oh indeed, that's true. Let's keep it then, you might need it in the future.